### ISSUE TITLE
[BUG]: Self-hosted version can't be used when following the instructions as documented.

### ISSUE THREAD

**Issue by m0un10** [getlago/lago#109]
**Describe the bug**

When following the instructions, it's not possible to create an API key.

**To Reproduce**
1. Following instructions [here](https://doc.getlago.com/docs/guide/self-hosting/docker)
2. After the docker-compose is up, go to `http://localhost`
3. It takes you to a login page.

**Expected behavior**

The documentation says after `docker-compose up`, `Your API Key can be found directly in the UI`. But instead there is a login screen and the title of page is `Lago Cloud`. I would expect that I would be able to following the docs and be able to get an API key as that is the expectation set in the documentation.

**Screenshots**

<img width="920" alt="Screen Shot 2022-09-25 at 12 13 46 pm" src="https://user-images.githubusercontent.com/7553951/192125306-d279a748-4d71-4d31-b9db-24a2a07ddd19.png">

**Support**
 - OS: Mac
 - Browser `Chrome 105.0.5195.125 (Official Build) (x86_64)`, Safari Version 16.0 (17614.1.25.9.10, 17614)`

```
cat docker-compose.yml | grep image
    image: postgres:14.0-alpine
    image: redis:6.2-alpine
    image: getlago/api:v0.10.0-alpha
    image: getlago/front:v0.10.0-alpha
    image: certbot/certbot
    image: getlago/api:v0.10.0-alpha
    image: getlago/api:v0.10.0-alpha
    image: getlago/lago-gotenberg:7
```

**Additional context**

I've seen other examples have a `Sign Up` link on the main page, but I don't have that. Calling it out in case that is the issue. However, I'm not sure since the docs don't mention anything about `Sign Up`, just to go to the UI for API key.

**Comment by sarkissianraffi** [comment]
Hi @m0un10, 
This is not related to a bug. I've updated the [documentation](https://doc.getlago.com/docs/guide/self-hosting/docker#signing-up) consequently.
It's mandatory to sign up your organization when using Lago with the self hosted version. The reason is that this organization is the core piece of the billing. It will be used to invoice your customers, invite other team members, create plans and billable metrics.

If you already have an account, you can just log yourself in.

I'm closing this issue with this comment, but feel free to follow up on this.

Raffi

**Comment by promentol** [comment]
I also don't see sign up link
from codebase I found it, /sign-up, but it's showing Page not found
I have verified, LAGO_DISABLE_SIGNUP=false, in my env variables, in docker container.
Tried on latest codebase from this repository

What can we do?

**Comment by sarkissianraffi** [comment]
@promentol, we have solved similar issues for other users. If you join our [Slack community](https://join.slack.com/t/lago-community/shared_invite/zt-1br015s0y-y_ItVfZrjkNsf9OHY_eUJw) you can jump directly into this https://lago-community.slack.com/archives/C03MCH55EL8/p1663583396632829

**Comment by jdenquin** [comment]
@promentol @m0un10 it's fixed on the last lago's front version `v0.10.1-alpha` 
You can just pull the lago repo and restart all you docker containers!
